features/fixtures/app/module/config/bugsnag/all_disabled.gradle

```diff
@@ -0,0 +1,6 @@
+project.afterEvaluate {
+    project.bugsnag.endpoint = "http://localhost:9339"
```

Comment:
@fractalwrench Several of these configs seem to be additions rather than updates.  I'm not entirely sure where they've come from, but if you can confirm they aren't needed I'll remove them.

Reply:
As part of v5 I believe the directory containing these snippets was moved to `features/fixtures/app/config` - so these should be safe to remove